pkg/solana/chainwriter/ccip_example_config.go

```diff
@@ -1,28 +1,42 @@
 package chainwriter
 
 import (
+	"context"
+	"fmt"
+	"reflect"
+
+	ag_binary "github.com/gagliardetto/binary"
 	"github.com/gagliardetto/solana-go"
 )
 
+const registryAddress = "4Nn9dsYBcSTzRbK9hg9kzCUdrCSkMZq1UR6Vw1Tkaf6A"
+
+// Should match type here: https://github.com/smartcontractkit/chainlink-ccip/blob/6bb9583526ce7c281005d66ffdf4a5300fc8ddc9/chains/solana/gobindings/ccip_router/accounts.go#L707-L713
+type TokenAdminRegistry struct {
+	Version              uint8
+	Administrator        solana.PublicKey
+	PendingAdministrator solana.PublicKey
+	LookupTable          solana.PublicKey
+	WritableIndexes      [2]ag_binary.Uint128
+}
+
 func TestConfig() {
```

Comment:
Have you run this local against the solana contracts?

Reply:
I have not yet - this is an example for the plugins team to use to port over for use in the E2E tests and in the production plugin code. I thought that due to the complexity of setting up the CCIP Solana contracts and running tests against them using this config it wouldn't make sense to run this config locally, instead we would rely on the E2E tests to validate this config in a real environment